2.0.1/CHANGELOG.md

```diff
@@ -0,0 +1,300 @@
+# Changelog
+
+Astronomer Certified 2.0.1-1 2020-01-21
```

Comment:
We don't add the entire Airflow changelog here, we only add what is new compared to Airflow 2.0.1

Example: https://github.com/astronomer/ap-airflow/blob/master/2.0.0/CHANGELOG.md#astronomer-certified-200-1-2020-12-17

Reply:
Also check the format of the changelog